pipeline/tests/asm.rs

```diff
@@ -2,8 +2,8 @@ use powdr_number::{FieldElement, GoldilocksField};
 use powdr_pipeline::test_util::{gen_estark_proof, test_halo2, verify_test_file};
 use test_log::test;
 
-fn verify_asm<T: FieldElement>(file_name: &str, inputs: Vec<T>) {
-    verify_test_file(file_name, inputs, vec![]);
+fn verify_asm(file_name: &str, inputs: Vec<GoldilocksField>) {
```

Comment:
feels actually odd we don't test BN here at all

Reply:
I agree, but it was like that before.